src/eos_stream.c

```diff
@@ -27,14 +27,20 @@
 #include "eos_parse_eosio.h"
 #include "eos_parse_unknown.h"
 
-#define EOSIO_TOKEN          0x5530EA033482A600
+#define EOSIO_TOKEN        0x5530EA033482A600
+#define CORE_VAULTA        0x452EA06CDA8E4C00
+
 #define EOSIO_TOKEN_TRANSFER 0xCDCD3C2D57000000
```

Comment:
I think we should rename this to `TRANSFER`  because it is not eosio specific.

Reply:
![Screenshot 2025-05-14 at 2 13 22 PM](https://github.com/user-attachments/assets/e80c6b9d-2aea-4458-ac7d-d8369a7c839c)

Agreed Renamed